source/extensions/omni.isaac.lab/omni/isaac/lab/app/app_launcher.py

```diff
@@ -271,6 +275,15 @@ def add_app_launcher_args(parser: argparse.ArgumentParser) -> None:
                 " it is resolved relative to the `apps` folder in Isaac Sim and Isaac Lab (in that order)."
             ),
         )
+        arg_group.add_argument(
+            "--ov_args",
+            type=str,
+            default="",
```

Comment:
Can we do something better than a string here?

https://stackoverflow.com/questions/15753701/how-can-i-pass-a-list-as-a-command-line-argument-with-argparse

```suggestion
            type=str,
            default="",
```